pyasdf/tags/transform/selector.py

```diff
@@ -0,0 +1,77 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, division, unicode_literals, print_function
+
+import numpy as np
+from numpy.testing import assert_array_equal
+
+
+from gwcs import selector
```

Comment:
I agree this is not the place for these types (I was simply trying to test them).
I guess they can go in astropy eventually but I'm trying to avoid this for the time being (until we have some more experience).
I'm aslo assuming there will be a `gwcs.WCS.toasdf()` method (I may be wrong) in which case 1)  and 2) lead to circular imports, right?
How would 4) work in practice? Is it ready to be tested?